apisix/plugins/error-log-logger.lua

```diff
@@ -180,10 +180,15 @@ local function send_to_skywalking(log_message)
     httpc:set_timeout(config.timeout * 1000)
 
     local entries = {}
+    local serviceInstanceName = config.skywalking.service_instance_name
```

Comment:
IMO, we should use this code style, like "service_instance_name`

Reply:
Yes, we should use snake case naming